scene/gui/control.cpp

```diff
@@ -2158,6 +2158,11 @@ Control::FocusBehaviorRecursive Control::get_focus_behavior_recursive() const {
 	return data.focus_behavior_recursive;
 }
 
+bool Control::_is_focusable() const {
+	bool ac_enabled = is_inside_tree() && get_tree()->is_accessibility_enabled();
+	return (is_visible_in_tree() && ((get_focus_mode_with_override() == FOCUS_ALL) || (get_focus_mode_with_override() == FOCUS_CLICK) || (ac_enabled && get_focus_mode_with_override() == FOCUS_ACCESSIBILITY)));
```

Comment:
I didn't notice earlier, but allowing FOCUS_CLICK here is wrong and it wasn't there before. E.g. `find_next_valid_focus()` is used for joypad focus, while this mode should only allow mouse button focus.

Reply:
`find_next_valid_focus()` is documented as `Finds the next [Control] that can receive the focus.`, and was using `!= FOCUS_NONE` before, so it should handle `FOCUS_CLICK` exactly the same as it was.

Reply:
Ah right .-.